app/controllers/application_controller.rb

```diff
@@ -1,4 +1,12 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
would suggest generating devise registration controller and place this logic there